### DISCUSSION TITLE
Is it possible to set layout to an Asyncsite?

### DISCUSSION THREAD

**Discussion by a-ghorbani** [h2oai/wave#2110] [Q&A] [answered]
Can we configure the layout for an AsyncSite? I've been unable to find an example in the documentation.

Is the this code a correct way of do it?
```python
site = AsyncSite()
page = site["/mysite"]
page["meta"] = ui.meta_card(
        box="",
        theme="h2o-dark",
        layouts=[ui.layout(...)...])
```

**Comment by mturoci** [comment]
You can configure everything like in any Wave app/script, including layouts. It's hard to say without seeing the full code, but my guess is you forgot to call the save:

```py
site = AsyncSite()
page = site["/mysite"]
page["meta"] = ui.meta_card(
        box="",
        theme="h2o-dark",
        layouts=[ui.layout(...)...])
await page.save()
```

**Reply by a-ghorbani** [reply, marked as answer]
My bad. The issue was that I mistakenly used page.drop() to remove the cards. I didn't notice that 'meta' is also a card that would be removed as well.
